### PR TITLE
test and fix master

### PR DESCRIPTION
**Pull request by chriddyp**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -58,3 +58,13 @@ test:
     # since optional requirements is part of the test suite, we don't need to
     # worry about testing that it *can* be imported in this case.
     - $(! python -c "import plotly.figure_factory")
+
+    # test the distribution and a basic import
```
Comment (cldougl) [inline circle.yml]: 👍 good idea

**Comment by chriddyp** [comment]
ready for review @cldougl and @Kully

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+## Fixed
+- `import plotly` was broken in `2.0.3` and `2.0.2` because the new `dashboard_objs` wasn't included in our `setup.py`'s "`packages`". Now it is and `import plotly` and the other features introduced in `2.0.3` and `2.0.2` should work.
+
+## [2.0.3] - 2017-03-06 [YANKED]
 ## Added
 - Dashboards can now be created using the API and uploaded to Plotly. Use `import plotly.dashboard_objs` to create a `Dashboard` object. You can learn more about `Dashboard` objects by running `help(plotly.dashboard_objs)` and `help(plotly.plotly.plotly.dashboard_ops)` for uploading and retrieving dashboards from the cloud.
 
 
-## [2.0.2] - 2017-02-20
+## [2.0.2] - 2017-02-20 [YANKED]
```
Comment (cldougl) [inline CHANGELOG.md]: 2.0.2 was the last stable version

2.0.3 and 2.0.4 were yanked and we'll need to release 2.0.5

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,12 +4,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-## [2.0.3] - 2017-03-06
+## [2.0.4] - 2017-03-07
```
Comment (cldougl) [inline CHANGELOG.md]: so this should be 2.0.5

**Comment by cldougl** [comment]
💃 after straightening out changelog

**Comment by Kully** [comment]
💃

**Pull request merged**